documentation/helpers/ToC/utils.js

```diff
@@ -0,0 +1,11 @@
+export function scrollToAnchor(anchorId) {
+  const element = document.getElementById(anchorId)
+  if (element) {
+    element.scrollIntoView({ behavior: 'smooth' })
+  }
+}
+
+export function updateUrlAnchor(hash) {
+  // eslint-disable-next-line no-undef
+  parent.location.href = parent.location.href.split('#')[0] + '#' + hash
```

Comment:
Since this function performs side effects, it may be better to avoid this function and instead have the side effects directly at the call site where they are not hidden. This could avoid any potential confusion 🤷